src/data/utils.ts

```diff
@@ -0,0 +1,5 @@
+import {Graph} from '../logic/commonInterfaces';
```

Comment:
I think this file would fit more in `logic` folder.

Reply:
I'd leave it here since it is used  only with the samples